src/solc/SolcVersions.ts

```diff
@@ -0,0 +1,32 @@
+import { BuidlerPluginError } from "@nomiclabs/buidler/plugins";
+import request from "request-promise";
+
+export default class SolcVersions {
+  public static async toLong(shortVersion: string) {
+    const solc = new SolcVersions();
+    return solc.getLongVersion(shortVersion);
+  }
+
+  public async getLongVersion(shortVersion: string) {
+    const versions = await this.getVersions();
+    const fullVersion = versions[shortVersion];
+    if (!fullVersion) {
+      throw new BuidlerPluginError("Given solc version doesn't exists");
+    }
+    return fullVersion.replace(/(soljson-)(.*)(.js)/, "$2");
+  }
+
+  public async getVersions(): Promise<{ [key: string]: string }> {
+    try {
+      const response: any = await request.get(
+        "https://raw.githubusercontent.com/ethereum/solc-bin/gh-pages/bin/list.json",
```

Comment:
We have a this typed in Buidler's core. But it's not easy to share that at the moment. Will change it when we finish the migration into a monorepo.

Reply:
I went to use CompilerDownloader but it messes up tests becuase it's caching list.json locally. Would be nice if it provided a flag to prevent caching